tests/test_gndtruth.bats

```diff
@@ -0,0 +1,122 @@
+################################################################################
+# Unpacks a gzip file                                                          #
+# ---------------------------------------------------------------------------- #
+#   Parameter 1: extension of the filename (after /tmp/cloe_gndtruth.)         #
+#  Precondition: gzip file /tmp/cloe_gndtruth.$1 exists                        #
+# Postcondition: /tmp/cloe_gndtruth.*.$1 is unpacked                           #
+################################################################################
+
+fn_unpack_gz() {
+    gunzip /tmp/cloe_gndtruth.$1
+}
+
+################################################################################
+# fn_unpack_bz2                                                                #
+################################################################################
+# Unpacks a bzip2 file                                                         #
+# ---------------------------------------------------------------------------- #
+#   Parameter 1: extension of the filename (after /tmp/cloe_gndtruth.)         #
+#  Precondition: bzip2 file /tmp/cloe_gndtruth.$1 exists                       #
+# Postcondition: /tmp/cloe_gndtruth.*.$1 is unpacked                           #
+################################################################################
+
+fn_unpack_bz2() {
+    bunzip2 /tmp/cloe_gndtruth.$1
+}
+
+################################################################################
+# fn_test_gndtruth                                                             #
+################################################################################
+# Minimal test for properties of a gndtruth file                               #
+# ---------------------------------------------------------------------------- #
+#   Parameter 1: extension of the filename (after /tmp/cloe_gndtruth.)         #
+#  Precondition: bzip2 file /tmp/cloe_gndtruth.$1 exists                       #
+# Postcondition: /tmp/cloe_gndtruth.*.$1 was tested                            #
+#          Note: Tested is occurrence of the gndtruth_lane_sensor              #
+#          Note: Tested is occurrence of the default_world_sensor              #
+#          Note: EXPECTED_CYCLE_COUNT := 1s / 20ms/Cycle - 1 Cycle             #
+################################################################################
+
+fn_test_gndtruth() {
+    EXPECTED_CYCLE_COUNT=49;
+    ACTUAL_CYCLE_COUNT=$(cat /tmp/cloe_gndtruth.$1 | grep -ao "cloe::gndtruth_lane_sensor" | wc -l)
+    if [[ "$ACTUAL_CYCLE_COUNT" != "$EXPECTED_CYCLE_COUNT" ]]; then
+        exit 1;
+    fi;
+    ACTUAL_CYCLE_COUNT=$(cat /tmp/cloe_gndtruth.$1 | grep -ao "cloe::default_world_sensor" | wc -l)
+    if [[ "$ACTUAL_CYCLE_COUNT" != "$EXPECTED_CYCLE_COUNT" ]]; then
+        exit 1;
+    fi;
+    fn_cleanup
+}
+
+@test "Expect json.gz : test_gndtruth_smoketest-json-gz.json (Test-ID b3a11bb5-6fd1-401e-b9fb-db76b4c338e5)" {
```

Comment:
What's up with the "Test-ID" here?

Reply:
A requirement of the accompanying documentation is to link text and implementation logically using a GUID.